resources/services/emr/block_public_access_configs.go

```diff
@@ -110,6 +110,10 @@ func fetchEmrBlockPublicAccessConfigs(ctx context.Context, meta schema.ClientMet
 		options.Region = c.Region
 	})
 	if err != nil {
+		if client.IgnoreNotAvailableRegion(err) {
+			meta.Logger().Debug("received InvalidRequestException on GetBlockPublicAccessConfiguration, api is not available in the current Region.", "err", err)
```

Comment:
maybe log region as well?

Reply:
@disq - Could we add `region` as a default value that is included in all log messages?

Reply:
I am not sure the region is needed at all as it is already in all log statements: https://github.com/cloudquery/cq-provider-aws/blob/c1269e14ee46e00df1f3793e440a3b40214a7ef6/client/client.go#L262-L275